website/content/docs/agent/options.mdx

```diff
@@ -1396,13 +1396,17 @@ bind_addr = "{{ GetPrivateInterfaces | include \"network\" \"10.0.0.0/8\" | attr
       certificate will be requested by a proxy before this limit is reached. This
       is also the effective limit on how long a server outage can last (with no leader)
       before network connections will start being rejected. Defaults to `72h`.
-      This value cannot be lower than 1 hour or higher than 1 year.
+      This value cannot be lower than 1 hour or higher than 1 year. Must use hour
+      units or smaller, for example, `1d` is not valid but `24h` is.
```

Comment:
```suggestion
      This value cannot be lower than 1 hour or higher than 1 year. Must use hour
      units or smaller, for example, `1d` is not valid but `24h` is.
```
I'm confused -- does this param only accept values measured in `h`? If so can we just say it that way and that the minimum is `1h` and the maximum is `8760h`?

Reply:
The supported time format values are specified here <https://www.consul.io/docs/agent/options#config_key_reference>. `h`[our] is the largest unit supported by Go's `time.ParseDuration` function.

Reply:
Given that ☝️ how would you like this worded @trujillo-adam ?